cluster-autoscaler/cloudprovider/azure/azure_config.go

```diff
@@ -128,6 +131,9 @@ type Config struct {
 	// Jitter in seconds subtracted from the VMSS cache TTL before the first refresh
 	VmssVmsCacheJitter int `json:"vmssVmsCacheJitter" yaml:"vmssVmsCacheJitter"`
 
+	// GetVmssSizeRefreshPeriod (seconds) defines how frequently to call GET VMSS API to fetch VMSS info per nodegroup instance
+	GetVmssSizeRefreshPeriod int `json:"getVmssSizeRefreshPeriod" yaml:"getVmssSizeRefreshPeriod"`
```

Comment:
Looks like I missed this, but `GetVmssSizeRefreshPeriod` being an `int` introduces an inconsistency with the fork, where it is `time.Duration`. This time it is not too bad, but let us all be careful next time. I could see cases like this getting worse if there are more dependencies.